extras/rya.pcj.fluo/pcj.fluo.app/src/main/java/org/apache/rya/indexing/pcj/fluo/app/query/CommonNodeMetadata.java

```diff
@@ -99,4 +99,18 @@ public String toString() {
                 .append("}")
                 .toString();
     }
+    
+    /**
+     * Base interface for all metadata Builders.  Using this type def
+     * allows for the implementation of a Builder visitor for navigating
+     * the Builder tree.
+     *
+     */
+    public static interface Builder {
```

Comment:
That's because it a type def that is meant to be extended.  If a build method was added, it would have to pass back a type, which would make it necessary to cast for extensions of this class that pass back extensions of CommonNodeMetadata.